cmd/prometheus/main.go

```diff
@@ -421,6 +422,9 @@ func main() {
 	serverOnlyFlag(a, "rules.max-concurrent-evals", "Global concurrency limit for independent rules that can run concurrently.").
 		Default("4").Int64Var(&cfg.maxConcurrentEvals)
 
+	serverOnlyFlag(a, "rules.evaluation-delay", "Duration to delay the evaluation of rules to ensure the underlying metrics have been received. Metric availability delays are more likely to occur when Prometheus is running as a remote write target, but can also occur when there's anomalies with scraping.").
```

Comment:
To make it extra clear that this is a one-time thing, I'd change "Duration to delay the evaluation..." to "Duration to delay the initial evaluation...", or just "Initial delay before starting rule evaluation...".

Reply:
Done in https://github.com/prometheus/prometheus/pull/14061/commits/c2f161ef3592920f8ddfd1223978baf0021f7339